debian/rules

```diff
@@ -88,6 +88,7 @@ endif
 	    -DWITHOUT_TOKUDB=true \
 	    -DWITHOUT_CASSANDRA=true \
 	    -DPLUGIN_AWS_KEY_MANAGEMENT=NO \
+	    -DWITH_INNODB_SNAPPY=ON \
```

Comment:
As @dr-m stated [here](https://jira.mariadb.org/browse/MDEV-11068?focusedCommentId=150907&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-150907), there should be some strong arguments before enabling a compression algorithm for a storage engine, I honestly don't know if fixing the issue stated in this PR is one such strong argument, someone with a broader view over this whole issue should give the final OK for this PR.

Reply:
I have requested review from @vuvova and @dr-m. They should make this call for upstream. In Debian we support Snappy and it is a build dependency both upstream and in Debian. If upstream does not want to support Snappy, the build dep should be removed and Debian will then continue with this as an explicit diversion not intended for upstreaming. So far I've had the opinion that this is a useful "fix" to upstream.